lib/smart_answer_flows/marriage-abroad/outcomes/_services_and_fees.govspeak.erb

```diff
@@ -7,34 +7,34 @@
              collection: calculator.services,
              as: :service,
              locals: { calculator: calculator } %>
-<% end %>
 
-<% unless calculator.country_without_consular_facilities? ||
-   calculator.ceremony_country == 'indonesia' ||
-   calculator.ceremony_country == 'monaco' ||
-   calculator.ceremony_country == 'france' ||
-   calculator.ceremony_country == 'new-caledonia' ||
-   calculator.ceremony_country == 'wallis-and-futuna' ||
-   calculator.ceremony_country == 'french-guiana' ||
-   calculator.ceremony_country == 'french-polynesia' ||
-   calculator.ceremony_country == 'guadeloupe' ||
-   calculator.ceremony_country == 'martinique' ||
-   calculator.ceremony_country == 'mayotte' ||
-   calculator.ceremony_country == 'reunion' ||
-   calculator.ceremony_country == 'st-pierre-and-miquelon' %>
+  <% unless calculator.country_without_consular_facilities? ||
+     calculator.ceremony_country == 'indonesia' ||
+     calculator.ceremony_country == 'monaco' ||
+     calculator.ceremony_country == 'france' ||
+     calculator.ceremony_country == 'new-caledonia' ||
+     calculator.ceremony_country == 'wallis-and-futuna' ||
+     calculator.ceremony_country == 'french-guiana' ||
+     calculator.ceremony_country == 'french-polynesia' ||
+     calculator.ceremony_country == 'guadeloupe' ||
+     calculator.ceremony_country == 'martinique' ||
+     calculator.ceremony_country == 'mayotte' ||
+     calculator.ceremony_country == 'reunion' ||
+     calculator.ceremony_country == 'st-pierre-and-miquelon' %>
 
-  <% if calculator.ceremony_country == 'vietnam' %>
-    <%= render partial: 'link_to_consular_fees_pay_in_sterling_or_local_currency.govspeak.erb',
-               locals: { calculator: calculator } %>
+    <% if calculator.ceremony_country == 'vietnam' %>
+      <%= render partial: 'link_to_consular_fees_pay_in_sterling_or_local_currency.govspeak.erb',
+                 locals: { calculator: calculator } %>
 
-  <% elsif calculator.ceremony_country != 'cambodia' &&
-    !(calculator.ceremony_country == 'brazil' && calculator.resident_outside_of_third_country?) %>
-    <%= render partial: 'link_to_consular_fees.govspeak.erb',
-               locals: { calculator: calculator } %>
+    <% elsif calculator.ceremony_country != 'cambodia' &&
+      !(calculator.ceremony_country == 'brazil' && calculator.resident_outside_of_third_country?) %>
+      <%= render partial: 'link_to_consular_fees.govspeak.erb',
+                 locals: { calculator: calculator } %>
+    <% end %>
   <% end %>
-<% end %>
 
-<% if calculator.services.any? && calculator.services_payment_partial_name.present? %>
-  <% partial = 'payment_information_partials/' + calculator.services_payment_partial_name + '.govspeak.erb' %>
-  <%= render partial: partial, locals: { calculator: calculator } %>
+  <% if calculator.services_payment_partial_name.present? %>
```

Comment:
I've split this commit into two and reworded it to (hopefully) make things clearer:

> DRY up condition around rendering link to consular fees
> 
> I've moved the rendering of the link to consular fees within the outermost `if
> calculator.services.any?` condition. This allowed me to remove the duplicate
> `if calculator.services.any?` condition from the link_to_consular_fees partial.
> 
> Note that the link_to_consular_fees_pay_in_sterling_or_local_currency partial
> didn't have a similar condition. I believe this was because of an oversight in
> c2723341222c40d78156f4bdab246c448d6ac6a8. That commit describes renaming the
> consular_fees_in_cash_visa_master partial (which includes the condition) to
> link_to_consular_fees_pay_in_sterling_or_local_currency but that's not quite
> what it actually does.
> 
> This commit is best viewed using `--ignore-all-space`.

And 

> DRY up condition around rendering payment info
> 
> I've moved the rendering of the payment information within the outermost `if
> calculator.services.any?` condition. This allowed me to remove the duplicate
> `if calculator.services.any?` condition from the conditional surrounding the
> rendering of the payment information.
> 
> This commit is best viewed using `--ignore-all-space`.